Documentation/Manual/Texinfo/docstrings/macro-clouseau-formatting-place.texi

```diff
@@ -0,0 +1,19 @@
+@anchor{Macro clouseau:formatting-place}
+@deffn {Macro} {@symbol{formatting-place,clouseau}} object place-class cell present-place present-object &key place-var
+Execute @var{body} with @var{present-place} and @var{present-object}
+bound to print functions.
+
+Before executing @var{body}, an instance of @var{place-class}
+representing @var{container}'s child selected by @var{place-class} and
+@var{cell} is created and stored in the place associated with
+@var{container} unless such an instance already exists.
+
+@var{present-place} is bound to a function that, when called with a
+stream as its sole argument, outputs a presentation corresponding to the
+created place to the stream. The produced presentation will be of
+presentation-type @class{place}.
+
+@var{present-object} is bound to a function that, when called with a
+stream as its sole argument, outputs a presentation corresponding to
+@var{container}'s child selected by @var{place-class} and @var{cell}.
+@end deffn
```

Comment:
Not specific to this macro but consider providing examples where appropriate.

Reply:
Done.